docs/modules/core/pages/material/normal-types.adoc

```diff
@@ -0,0 +1,56 @@
+# Normal Map Conventions
+
+There are two conventions for normal maps:
+
+* OpenGL (known as OpenGL-Style)
+* DirectX (known as DirectX-Style)
```

Comment:
Could you please include Y+ and Y- as in the draft? These styles are frequently referenced this way, so having them at the start of the page would be a helpful addition